site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -783,15 +783,16 @@ the following properties:
 
 The browser handles sign-up and sign-in use cases differently depending on the
 existence of `approved_clients` in the response from [the accounts list
-endpoint](#accounts-list-endpoint). The browser will only display [the RP's
-privacy policy and terms of service](#client-metadata-endpoint) in the dialog if
-`approved_clients` isn't provided or does not include the RP's `clientId` and
-the user hasn't previously signed up for the RP in this browser.
+endpoint](#accounts-list-endpoint). The browser will display a disclosure text
+"To continue with ....". If `approved_clients` isn't provided or does not
+include the RP's `clientId` and the user hasn't previously signed up for the RP
+in this browser, the [privacy policy and terms of
+service](#client-metadata-endpoint) part will not be displayed.
```

Comment:
So, trying to understand this: If `approved_clients` is provided or if it
includes the RP's `clientId` and if the user has previously signed up for the RP
in this browser, the browser will display a text saying "To continue with ...." and privacy policy and terms of
service. If not, it will not have the privacy policy and terms of
service. Is that right?

Can we have the conditions and what is displayed on the screen listed out for clarity? Also what's a disclosure text? The only reference I could find was legal--is that what the reference is here?

Reply:
How about this:
```
The browser will display a disclosure text **"To continue with ...."**. If the user has already signed up to the RP, the [privacy policy and terms of service](/docs/privacy-sandbox/fedcm/#client-metadata-endpoint) part will be omitted. The sign-up state is determine by whether `approved_clients` includes the RP's `clientId` or if the browser remembers that the user has already signed up to the RP. 
```

Reply:
Makes sense, some nits: is determined*

Also, whether is used to present choices in sentences, if to provide conditions. So what it should be is something like:

The sign-up state is determined based on whether the following conditions are fulfilled or not:
* If `approved_clients` includes the RP's `clientId`.
* If the browser remembers that the user has already signed up to the RP.

Reply:
That makes sense. Applied the suggested change.